app/views/shared/_footer.html.erb

```diff
@@ -1,3 +1,4 @@
 <footer class="footer">
-  <p>&copy; entry lvl</p>
+    <svg class="footer__logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 704.15 121.28"><title>logo</title><g fill="#fff"><path d="M53.94 43.71q7.62 9.54 6.9 25.86H9.23a23.46 23.46 0 0 0 7.44 15 22.22 22.22 0 0 0 15.48 5.64 27.78 27.78 0 0 0 10.8-2.1 25.93 25.93 0 0 0 8.76-5.94l5.16 5.4a30.4 30.4 0 0 1-11.1 7.74 37.23 37.23 0 0 1-14.34 2.7 32.79 32.79 0 0 1-16.38-4 28.14 28.14 0 0 1-11.1-11.28A34.14 34.14 0 0 1 0 66.1a33.9 33.9 0 0 1 4-16.5 28.56 28.56 0 0 1 11.1-11.34 32.19 32.19 0 0 1 16.26-4.08q14.96-.01 22.58 9.53zm-1.5 18.39q-.48-9.24-6.18-14.7t-14.94-5.43q-9.09 0-15.09 5.46t-7 14.67zM128.4 40.71q6.36 6.54 6.36 18.06v38.88h-9.36V61.17q0-8.4-4.74-13.2t-13.14-4.8q-9.24.12-14.82 5.7t-6.47 14.94v33.84h-9.31V34.53h9.31v13.92q6.48-14 24.48-14.28 11.33 0 17.69 6.54zM188.23 93.33a26 26 0 0 1-14.52 4.68q-7.2 0-11.52-4.5t-4.27-13.41V43.77h-9.12v-7.32h9.12V19.89h9.24v16.56h20v7.32h-20v34.8q0 5.64 2.1 8.22t6.3 2.58a22.2 22.2 0 0 0 10.08-3.27zM220.68 37.95a26 26 0 0 1 13.8-3.78v9.12q-9.84-.12-15.84 5.46t-6.84 15.3v33.6h-9.36V34.53h9.36v14a23.65 23.65 0 0 1 8.88-10.58zM262.8 117.57a17.71 17.71 0 0 1-11.16 3.72 20.54 20.54 0 0 1-13.32-4.56l4-7.44a15 15 0 0 0 8.88 3 10.42 10.42 0 0 0 6.66-2.34 17.15 17.15 0 0 0 4.86-6.78l2.64-6-26.2-62.64H249l21 52.56 20-52.56h9.6l-29 72.12a26.16 26.16 0 0 1-7.8 10.92zM347.23 8.61h9.36v89h-9.36zM369.12 34.53h10l20.11 53.64 20-53.64h9.48l-24.84 63.12h-9.64zM440.39 8.61h9.36v89h-9.36z"/></g><g fill="#fff"><path d="M495.32 73.77h144.15V96.6H495.32zM527.66 36.99h144.15v22.83H527.66zM560 0h144.15v22.83H560z"/></g></svg>
```

Comment:
Can this SVG be moved to a partial? What about using [svgo](https://jakearchibald.github.io/svgomg/) to optimize it?

Reply:
+1 pls